actions/image_partition_action.go

```diff
@@ -628,6 +685,18 @@ func (i *ImagePartitionAction) Verify(context *debos.DebosContext) error {
 			return fmt.Errorf("Can only set partition partlabel on GPT filesystem")
 		}
 
+		if p.PartUUID != "" {
```

Comment:
Check Len > 0

Reply:
Thanks for your comment. I will revise it.